src/interpreter/plugin/RomanPlugin.ts

```diff
@@ -0,0 +1,180 @@
+/**
+ * @license
+ * Copyright (c) 2020 Handsoncode. All rights reserved.
+ */
+
+import {CellError, ErrorType, InternalScalarValue, SimpleCellAddress} from '../../Cell'
+import {ErrorMessage} from '../../error-message'
+import {ProcedureAst} from '../../parser'
+import {ArgumentTypes, FunctionPlugin} from './FunctionPlugin'
+
+export class RomanPlugin extends FunctionPlugin {
+  public static implementedFunctions = {
+    'ROMAN': {
+      method: 'roman',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER, minValue: 1, lessThan: 4000 },
+        { argumentType: ArgumentTypes.NOERROR, optionalArg: true, defaultValue: 0 }
+      ],
+    },
+    'ARABIC': {
+      method: 'arabic',
+      parameters: [
+        { argumentType: ArgumentTypes.STRING},
+      ],
+    },
+  }
+
+  public roman(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('ROMAN'),
+      (val: number, mode: InternalScalarValue) => {
+        val = Math.trunc(val)
+        if(mode === false) {
+          mode = 0
+        } else if(mode === true) {
+          mode = 4
+        }
+        mode = this.coerceScalarToNumberOrError(mode)
+        if(mode instanceof CellError) {
+          return mode
+        }
+        mode = Math.trunc(mode)
+        if(mode < 0) {
+          return new CellError(ErrorType.VALUE, ErrorMessage.ValueSmall)
+        }
+        if(mode > 4) {
+          return new CellError(ErrorType.VALUE, ErrorMessage.ValueLarge)
+        }
+        return romanMode(val, mode)
+      }
+    )
+  }
+
+  public arabic(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('ARABIC'),
+      (input: string) => {
+        input = input.trim().toUpperCase()
+        let minusSign = false
+        if(input.startsWith('-')) {
+          input = input.slice(1)
+          minusSign = true
+          if(input==='') {
+            return new CellError(ErrorType.VALUE, ErrorMessage.Roman)
+          }
+        }
+        let ans = 0;
+        [input, ans] = eatToken(input, ans,
+          {token: 'MMM', val: 3000},
+          {token: 'MM', val: 2000},
+          {token: 'M', val: 1000},);
+        [input, ans] = eatToken(input, ans,
+          {token: 'IM', val: 999},
+          {token: 'VM', val: 995},
+          {token: 'XM', val: 990},
+          {token: 'LM', val: 950},
+          {token: 'CM', val: 900},);
```

Comment:
Either ending comma or parenthesis. I would move it to the next line. Please check the whole file
```suggestion
          {token: 'CM', val: 900},
        );
```